tap_purecloud/__init__.py

```diff
@@ -496,6 +496,7 @@ def sync_management_units(api_client: ApiClient, config):
 
     # first, write out the units
     mgmt_units = stream_results(gen_units, ('entities', ), lambda x: x, 'management_unit', schemas.management_unit, ['id'], True)
+    should_output_schema = True
```

Comment:
Can we move this to line 522 and add a comment if its only needed for those two streams?

Reply:
I think we need to define this outside of the for loop to make sure that it tracks properly (since the bug was coming from when we hit the if block with the continue inside of the loop). And yes good call, I can definitely add a comment!